src/track/trace_libunwind.cpp

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
Should I put any copyright notice here? If so, what it should be exactly?

Reply:
the same as for the trace.h header but with an updated year (it's all my code basically)